google/ggrc-core
# GGRC-4211/GGRC-7189/GGRC-7190 Add proposals for program

# Issue description

User should be able to Propose changes to a Program

# Steps to test the changes

1. Create program
2. Open program
Expected result:
User should be able to add Proposal

# Solution description

Add proposable mixin to Program

# Sanity checklist




# PR Review checklist
